Library/Homebrew/global.rb

```diff
@@ -56,6 +56,10 @@
   %r[https://github\.com/([\w-]+)/([\w-]+)?/(?:pull/(\d+)|commit/[0-9a-fA-F]{4,40})]
 HOMEBREW_BOTTLES_EXTNAME_REGEX = /\.([a-z0-9_]+)\.bottle\.(?:(\d+)\.)?tar\.gz$/
 
+NEWEST_HOMEBREW_CORE_MACOS_RUNNER = :sequoia
+OLDEST_HOMEBREW_CORE_MACOS_RUNNER = :ventura
+NEWEST_HOMEBREW_CORE_INTEL_MACOS_RUNNER = :sonoma
```

Comment:
This is kinda weird global, especially if it's only for tests. I think the tests should just do `GitHubRunnerMatrix::NEWEST_HOMEBREW_CORE_MACOS_RUNNER`

Reply:
Sure, works for me. Just thought these values would be easier for other maintainers to tweak if they lived in global.